web/modules/custom/collection/collection.behat.inc

```diff
@@ -23,6 +25,8 @@ class CollectionSubContext extends DrupalSubContextBase implements DrupalSubCont
   use FileTrait;
   use RandomGeneratorTrait;
   use RdfEntityTrait;
+  use NodeTrait;
+  use OgMenuTrait;
```

Comment:
It would be so nice if reformatting in PhpStorm could do this automatically as it does I think with global use statements. :)